charmhelpers/contrib/openstack/context.py

```diff
@@ -1748,9 +1748,9 @@ def __init__(self, name=None, script=None, admin_script=None,
 
     def __call__(self):
         total_processes = _calculate_workers()
-        enable_wsgi_rotation = config('wsgi-rotation')
-        if enable_wsgi_rotation is None:
-            enable_wsgi_rotation = True
+        enable_wsgi_socket_rotation = config('wsgi-socket-rotation')
```

Comment:
This causes a potentially breaking change, albeit only on master branch charms.  Please could you link to the bug or gerrit reviews where this will be consumed?  Thanks.